pages/Hypr Ecosystem/hyprsunset.md

```diff
@@ -21,3 +26,25 @@ This method is preferred to screen shaders as it will _not_ be captured via reco
 ## Usage
 
 See `hyprsunset --help`.
+
+## IPC
+
+`hyprsunset` supports IPC via hyprctl. Both color temperature and the gamma filter are adjustable:
+```sh
+# Enable blue-light filter
+hyprctl hyprsunset temperature 2500
+# Disable blue-light filter
+hyprctl hyprsunset identity
+
+# Set gamma to 50%
+hyprctl hyprsunset gamma 50
+# Increase gamma by 10%
+hyprctl hyprsunset gamma +10
+```
+
+This can be used by other software to change the temperature throughout the day, or to adjust perceieved
```

Comment:
please note that this should be used as a last resort if stuff like `brightnessctl` or `light` don't work as it degrades color accuracy

Reply:
Added a warning callout. Personally, I have gamma control bound to shift+brightness keys, as well as normal software brightness, so that I can turn the brightness of my monitor down to its minimum, then reduce it further (especially at night) by decreasing the gamma. I'm not sure it necessarily has to be a "last resort", as long as you understand the difference.